<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WDUserSettings</class>
 <widget class="QDialog" name="WDUserSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户设定</string>
  </property>
  <widget class="QPushButton" name="btnFinish">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>完成</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pPwd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>原密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdNew">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>新密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lUsername">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>新用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdNewRepeat">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>重复新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>信息修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lMail">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>新邮箱/新手机号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>将栏目留空代表不需要进行修改</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
